<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IASegmentadorAbdominalViewControls</class>
 <widget class="QWidget" name="IASegmentadorAbdominalViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>482</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelWarning">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
     </property>
     <property name="text">
      <string>Por favor seleccionar una imagen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>¿Qué órgano/s desea segmentar?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbTodos">
     <property name="text">
      <string>Todos</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbBazo">
     <property name="text">
      <string>Bazo</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbRinonIzquierdo">
     <property name="text">
      <string>Riñón izquierdo</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbVesicula">
     <property name="text">
      <string>Vesícula</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbEsofago">
     <property name="text">
      <string>Esófago</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbHigado">
     <property name="text">
      <string>Hígado</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbEstomago">
     <property name="text">
      <string>Estómago</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbPancreas">
     <property name="text">
      <string>Páncreas</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbDuodeno">
     <property name="text">
      <string>Duodeno</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="infolabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="buttonPerformImageProcessing">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="text">
      <string>Realizar segmentación</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
